packages/ui/src/components/Form/expression/expression.service.ts

```diff
@@ -72,9 +73,10 @@ export class ExpressionService {
     language: ICamelLanguageDefinition | undefined;
     model: Record<string, unknown> | undefined;
   } {
+    if (!isDefined(parentModel)) return { language: undefined, model: undefined };
```

Comment:
This line was added due to a bug when handling the `setHeaders` EIP, that caused the newly added header to be `null`, creating an error in the `CanvasForm` component